mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json

```diff
@@ -184,6 +184,16 @@
       "run": "after",
       "snippetPath": "create_get_job_execution_summary_function.sql",
       "fromModuleVersion": "mod-source-record-manager-3.4.0"
+    },
+    {
+      "run": "after",
+      "snippet": "ALTER TYPE job_execution_status ADD VALUE IF NOT EXISTS 'CANCELLED'",
+      "fromModuleVersion": "mod-source-record-manager-3.4.0"
+    },
+    {
+      "run": "after",
+      "snippet": "ALTER TYPE job_execution_ui_status ADD VALUE IF NOT EXISTS 'CANCELLED'",
```

Comment:
Hello @VRohach, please add ';' symbol at the end of the queries to fix their execution. Propose merging them into one line since they have the same purpose, in terms of changes to be introduced. Also, we don't need to change the "create_new_job_executions_table.sql" file because the value for status enums going to be added by scripts in the schema.json. Thank you!